test/Cli/Func.E2E.Tests/Commands/FuncInit/PythonInitTests.cs

```diff
@@ -0,0 +1,154 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Azure.Functions.Cli.E2E.Tests.Traits;
+using Azure.Functions.Cli.TestFramework.Assertions;
+using Azure.Functions.Cli.TestFramework.Commands;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Azure.Functions.Cli.E2E.Tests.Commands.FuncInit
+{
+    [Trait(WorkerRuntimeTraits.WorkerRuntime, WorkerRuntimeTraits.Python)]
+    public class PythonInitTests(ITestOutputHelper log) : BaseE2ETests(log)
+    {
+        [Theory]
+        [InlineData("")]
+        [InlineData("v1")]
+        [InlineData("v2")]
+        public void Init_Python_App_With_Supported_Model(string programmingModel)
```

Comment:
Given these tests are all in `PythonInitTests` can we remove PythonApp from all the test names?